model/test_results.go

```diff
@@ -194,8 +194,10 @@ func (t *TestResults) Append(ctx context.Context, results []TestResult) error {
 			for result := range resultChan {
 				data, err := bson.Marshal(result)
 				if err != nil {
+					// We should continue on marhsalling
+					// errors so we can get as many test
+					// results in s3 as possible.
 					catcher.Wrapf(err, "problem marshalling bson for test result '%s'", result.TestName)
```

Comment:
I actually meant `continue` here, because the data is most likely invalid/nil if `bson.Marshal` fails, so putting bad data in the S3 bucket won't be helpful (or might panic).

Reply:
Sorry my head is fried after grpc debugging, I forgot to add the continue lol